tools/dotcms-cli/cli/src/test/resources/resources-config.json

```diff
@@ -8,6 +8,9 @@
     },
     {
       "pattern": ".*\\.dll$"
+    },
+    {
+      "pattern": "(.*\\/)?build\\.properties$"
```

Comment:
You can use just "build\\.properties$"  as the file we want is always going to be in the top level of the resources. 
 From what I can tell from testing we don't need a ^ to indicate start of line, and the $ to indicate the end of the line is not strictly needed, but we can follow convention.  I can confirm that both patterns do work to fix the issue